src/main.rs

```diff
@@ -66,14 +70,18 @@ fn main() -> Result<()> {
     match opt {
         Opt::ListDevices => deviceinfo::list_devices(),
         Opt::ListKeys => list_keys(),
-        Opt::Remap { config_file } => {
+        Opt::Remap { config_file, delay } => {
             let mapping_config = MappingConfig::from_file(&config_file).context(format!(
                 "loading MappingConfig from {}",
                 config_file.display()
             ))?;
 
             log::error!("Short delay: release any keys now!");
-            std::thread::sleep(Duration::new(2, 0));
+            let float_delay_result = FloatDuration::seconds(delay).to_std();
+            match float_delay_result {
+                Ok(v) => std::thread::sleep(v),
+                Err(e) => log::error!("Invalid delay: {e:?}"),
+            }
```

Comment:
We can avoid the external dependency by using:

https://doc.rust-lang.org/std/time/struct.Duration.html#method.from_secs_f64

```suggestion
            std::thread::sleep(Duration::from_secs_f64(delay));
```